bpm/bonita-core/bonita-platform-login/bonita-platform-login-api-impl/src/test/java/org/bonitasoft/engine/core/platform/login/impl/PlatformLoginServiceImplTest.java

```diff
@@ -79,17 +81,18 @@ public final void loginThrowSInvalidUserException() throws SPlatformLoginExcepti
         platformLoginServiceImpl.login(userName, password);
     }
 
-    @Test(expected = SPlatformLoginException.class)
-    public final void loginThrowSInvalidPasswordException() throws SPlatformLoginException, SInvalidUserException, SInvalidPasswordException {
+    @Test(expected = SInvalidPlatformCredentialsException.class)
+    public final void loginThrowSInvalidPasswordException() throws SPlatformLoginException, SInvalidUserException, SInvalidPasswordException, SInvalidPlatformCredentialsException {

```

Comment:
Not sure to understand.
The behavior changed here to throw SInvalidPlatformCredentialsException instead of SPlatformLoginException.
But for other type of exception there is still only the SPlatformLoginException